utils/crypto/rsa2secp256k1.go

```diff
@@ -75,3 +76,13 @@ func RecoverSecp256PublicKey(cert *x509.Certificate) ([]byte, error) {
 
 	return hashing.PubkeyBytesToAddress(sPubKeyBytes), nil
 }
+
+func FakePrivateKey(addr ids.ShortID) *PrivateKeySECP256K1R {
```

Comment:
Doesn't look like it belongs to this file `rsa2secp256k1.go` judging by its name